### PR TITLE
v1.6.15-dev-config

### PR DESCRIPTION
**Pull request by iulianpascalau**
## Reasoning behind the pull request
- refactored configs
  
## Proposed changes
- 
- 
- 

## Testing procedure
- 
- 
- 

## Pre-requisites

Based on the [Contributing Guidelines](https://github.com/multiversx/mx-chain-go/blob/master/.github/CONTRIBUTING.md#branches-management) the PR author and the reviewers must check the following requirements are met:
- was the PR targeted to the correct branch?
- if this is a larger feature that probably needs more than one PR, is there a `feat` branch created?
- if this is a `feat` branch merging, do all satellite projects have a proper tag inside `go.mod`?

### CODE REVIEW

File: cmd/node/config/genesis.json [hunk 1]
```diff
@@ -1,92 +1,497 @@
...
+    "address": "erd1zed89c8226rs7f59zh2xea39qk9ym9tsmt4s0sg2uw7u9nvtzt3q8fdj2e",
+    "supply": "9509990000000000000000000",
+    "balance": "9509990000000000000000000",
+    "stakingvalue": "0",
+    "delegation": {
+      "address": "",
+      "value": "0"
     }
   }
 ]
```
Comment (sstanculeanu) [inline cmd/node/config/genesis.json:497]: missing new line

File: cmd/node/config/nodesSetup.json [hunk 2]
```diff
@@ -1,48 +1,395 @@
...
+    },
+    {
+      "info": "single key 24",
+      "pubkey": "0382c11222db8a15e42e3ff64893df46c7720b439fb2a546462815ac0a8fa3bed99fceae5da9b68524e36f61cc074d09ceafec274c54f182c56a77583f9421f19c777265c43da1d5747304b36f0367cf3e8e5f63f41dad1a4362d9e1997a9e16",
+      "address": "erd125eyrjk99zadr04gm9z2p4nckmnegexs5nyk7ek85rut2665t75sql3w88",
+      "initialRating": 5000001
     }
   ]
-}
+}
```
Comment (sstanculeanu) [inline cmd/node/config/nodesSetup.json:395]: missing new line

File: cmd/node/config/testKeys/validatorKey.pem [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+-----BEGIN PRIVATE KEY for 11f784d2970d65769ce267710b3d08b28b78c3f79283758918c8ef15717ccbe90c23348cafe0e98a5d101b8dafbe7d081c6821dee8bf40ba150664ccc2dbbdd6358c92404e677d82910ce61f1d7584fbbbc9ebf71b7f35a118556e2a5c220501-----
+MjU2ZGI2MmU3ZTBmMzkzMjlhYmM1YzE1NWM2NmE0YTdhNmRhOTY2MTVmMDgxOTMz
+NTYwMzU0YjllNWQ3YjYyYw==
+-----END PRIVATE KEY for 11f784d2970d65769ce267710b3d08b28b78c3f79283758918c8ef15717ccbe90c23348cafe0e98a5d101b8dafbe7d081c6821dee8bf40ba150664ccc2dbbdd6358c92404e677d82910ce61f1d7584fbbbc9ebf71b7f35a118556e2a5c220501-----
+-----BEGIN PRIVATE KEY for 0382c11222db8a15e42e3ff64893df46c7720b439fb2a546462815ac0a8fa3bed99fceae5da9b68524e36f61cc074d09ceafec274c54f182c56a77583f9421f19c777265c43da1d5747304b36f0367cf3e8e5f63f41dad1a4362d9e1997a9e16-----
+ZTUxOWQwNzcwZWRlZDhhNTFiMzIwN2M4MWRmMDhjMWZlMWZhMTQ1ZjFmYWQwNDU3
+YzI4NzRiNWQzYmY3Y2MwMw==
+-----END PRIVATE KEY for 0382c11222db8a15e42e3ff64893df46c7720b439fb2a546462815ac0a8fa3bed99fceae5da9b68524e36f61cc074d09ceafec274c54f182c56a77583f9421f19c777265c43da1d5747304b36f0367cf3e8e5f63f41dad1a4362d9e1997a9e16-----
```
Comment (sstanculeanu) [inline cmd/node/config/testKeys/validatorKey.pem:96]: missing new line

**Review by BeniaminDrasovean — APPROVED** [review]

**Review by gabi-vuls — DISMISSED** [review]

File: cmd/node/config/genesis.json (hunk 1 above)
Comment (iulianpascalau) [reply cmd/node/config/genesis.json:497]: done

File: cmd/node/config/nodesSetup.json (hunk 2 above)
Comment (iulianpascalau) [reply cmd/node/config/nodesSetup.json:395]: done

**Review by sstanculeanu — APPROVED** [review]

**Review by BeniaminDrasovean — APPROVED** [review]

**Review by gabi-vuls — APPROVED** [review]

**Pull request merged**